web/src/pages/api/debug/greptimedb-query.ts

```
@@ -0,0 +1,234 @@
+
+    let sqlQuery = "";
+    let queryParams: any[] = [];
+
+    switch (queryId) {
+      case "query1_complex_trace":
+        if (!tagsFilterValue || !metadataJsonPath || !metadataValue) {
+            return res.status(400).json({ error: "Missing parameters for Query 1 (tagsFilterValue, metadataJsonPath, metadataValue)." });
+        }
+        sqlQuery = QUERY_1_COMPLEX_TRACE;
+        // Construct the LIKE pattern carefully for tags
+        const tagsLikePattern = `%\"${tagsFilterValue}\"%`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)